doc/html/pcre2api.html

```diff
@@ -952,6 +957,110 @@ <h1>pcre2api man page</h1>
 nesting, and the second is user data that is set up by the last argument of
 <b>pcre2_set_compile_recursion_guard()</b>. The callout function should return
 zero if all is well, or non-zero to force an error.
+<br>
+<br>
+<b>int pcre2_set_optimize(pcre2_compile_context *<i>ccontext</i>,</b>
+<b>  uint32_t <i>directive</i>);</b>
```

Comment:
Maybe we should mention that we return an error if the `directive` is not known, or `ccontext` is NULL.

Reply:
I have added documentation mentioning that we return an error if `directive` is not known. Regarding `ccontext` being null, the other existing `pcre2_set_*` functions do not check for that. The caller is expected to ensure that `ccontext` is not NULL. And the documentation for other existing `pcre2_set_*` do not mention this.